blink/memorymalloc.c

```diff
@@ -67,7 +67,9 @@ static size_t GetBigSize(size_t n) {
 
 void FreeBig(void *p, size_t n) {
   if (!p) return;
+#if !defined(__EMSCRIPTEN__)
```

Comment:
This probably just needs a change to mirror the special behaviour in `AllocateBig` to just use the size as-is, i.e.

```
#if defined(__CYGWIN__) || defined(__EMSCRIPTEN__)
  unassert(!munmap(p, n));
#else
  unassert(!munmap(p, GetBigSize(n)));
#endif
```

Reply:
Somehow, this works! Included in the workaround.